src/main/java/com/uid2/admin/job/jobsync/keyset/ReplaceSharingTypesWithSitesJob.java

```diff
@@ -57,15 +57,15 @@ public ReplaceSharingTypesWithSitesJob(JsonObject config, WriteLock writeLock) {
 
         AdminKeysetStoreFactory adminKeysetStoreFactory = new AdminKeysetStoreFactory(
                 cloudStorage,
-                new CloudPath(config.getString(Const.Config.KeysetsMetadataPathProp)),
+                new CloudPath(config.getString("admin_keysets_metadata_path")),
```

Comment:
Should this be added to `Const.Config`?

Reply:
I would rather not do Const if we're not using this property elsewhere. Ideally this should be done with deserialization though

Reply:
Rather, should we remove everything from Const.Config that's only being used once?

Reply:
Not in the PR Jonathan